tests/test_lightning_sequence_labeling_pipeline.py

```diff
@@ -6,17 +6,30 @@
 import pytest
 import pytorch_lightning as pl
 
+from embeddings.pipeline.hf_preprocessing_pipeline import HuggingFacePreprocessingPipeline
 from embeddings.pipeline.lightning_pipeline import LightningPipeline
 from embeddings.pipeline.lightning_sequence_labeling import LightningSequenceLabelingPipeline
 
 
 @pytest.fixture(scope="module")
-def dataset_kwargs() -> Dict[str, Any]:
+def dataset_kwargs() -> Tuple[Dict[str, Any], "TemporaryDirectory[str]"]:
+    path = TemporaryDirectory()
+    pipeline = HuggingFacePreprocessingPipeline(
+        dataset_name="clarin-pl/kpwr-ner",
+        load_dataset_kwargs=None,
+        persist_path=path.name,
+        sample_missing_splits=None,
+        ignore_test_subset=False,
+        downsample_splits=(0.01, 0.01, 0.05),
+        seed=441,
+    )
+    pipeline.run()
+
     return {
-        "dataset_name_or_path": "clarin-pl/kpwr-ner",
+        "dataset_name_or_path": path.name,
         "input_column_name": "tokens",
         "target_column_name": "ner",
-    }
+    }, path
```

Comment:
do we need the path to be returned if later in the pipeline we use `dataset_kwargs[0]`?

Reply:
Same reason as previous